src/ex3/commands.js

```diff
@@ -0,0 +1,194 @@
+// imports
+import fs from 'fs';
+import { PokemonClient } from './pokemonClient.js';
+import { ItemManager } from './itemManager.js';
+import chalk from 'chalk';
+
+// PokemonClient instance
+const pokemonClient = new PokemonClient();
+const itemManager = new ItemManager();
+
+/* Add todo  function */
+const addTodo = async (todo) => {
+    let todoText = '';
+    let message = '';
+    if(itemManager.isPokemon(todo)){
+        const pokemon = await pokemonClient.getPokemonName(todo);
+        const type = await pokemonClient.getPokemonsTypes(pokemon);
+        todoText =`Catch ${pokemon}, a ${type} pokemon`;
+        message = `Catch ${chalk.bold(pokemon)}, a ${type} pokemon`;
+    } else{
+        todoText = todo;
+        message = "New todo added successfully!";
+    }
+    try{
+        fs.access('todos.txt', (err) => {
+            if (err){ 
+                // if todos.txt doesn't exist then create it
+                fs.writeFileSync('todos.txt', todoText);
+                console.log(chalk.bgGreen(message));
+            } else{
+                // else add todo to existiong file
+                try{
+                    fs.appendFileSync('todos.txt', `\n${todoText}`);
+                    console.log(chalk.bgGreen(message));
+                } catch{
+                    console.error(err);
+                }
+                let tasks = fs.readFileSync('todos.txt').toString().split('\n');
+            }
+        })
+    } catch(err){
+        console.error(err);
+    }
+}
+
+/* Get all the todos in todos.json */
+const getTodos = () => {
+    fs.access('todos.txt', (err) => {
+        if(err){
+            console.error(err);
+        } else{
+            console.log(chalk.underline.bgMagenta("All pending tasks"));
+            console.log(chalk.magenta(fs.readFileSync('todos.txt').toString()));
+            console.log(chalk.bold.bgRedBright(`There are ${fetchQuantity('todos.txt')} pending tasks`))
+        }
+    })
+}
+
+/* Get all the todos in todos.json */
+const getDoneTasks = () => {
+    fs.access('done.txt', (err) => {
+        if(err){
+            console.error(err);
+        } else{
+            console.log(chalk.underline.bgCyan("All done tasks:"))
+            console.log(chalk.cyan(fs.readFileSync('done.txt').toString()));
+            console.log(chalk.bold.bgRedBright(`There are ${fetchQuantity('done.txt')} done tasks`))
+        }
+    })
+}
+
+/* Deletes the todo with id todo */
+const deleteTodo = (todo) => {
+    const tasks = fs.readFileSync('todos.txt').toString().split('\n');
```

Comment:
What happens if the file doesn't exist?
![image](https://user-images.githubusercontent.com/12782209/173627683-427c28c7-ee73-4275-a4b8-942a7b943b9e.png)